src/Pos/Types/Block/Functions.hs

```diff
@@ -183,16 +186,20 @@ recreateMainBlock _gbHeader _gbBody _gbExtra = do
         Left err -> fail $ toString err
     pure gb
 
+-- | Dummy implementation of make genesis extra header data.
+mkGenesisEHD :: () -> GenesisExtraHeaderData
+mkGenesisEHD = GenesisExtraHeaderData . mkAttributes
```

Comment:
lol, this function will consume real attributes in the future. now it is only stub